scripts/train_agents.py

```diff
@@ -464,20 +79,40 @@ def N_X_SP(args) -> None:
         unseen_teammates_len=unseen_teammates_len,
         )
 
+def SPN_1ADV(args) -> None:
+    attack_rounds = 3
+    unseen_teammates_len = 1
+    adversary_play_config = AdversaryPlayConfig.MAP
+    primary_train_types = [TeamType.SELF_PLAY, TeamType.SELF_PLAY_ADVERSARY]
+
+    primary_eval_types = {'generate': [TeamType.SELF_PLAY_HIGH, TeamType.SELF_PLAY_MEDIUM, TeamType.SELF_PLAY_LOW, TeamType.SELF_PLAY_ADVERSARY],
```

Comment:
I usually don't include medium type for evaluation because the more eval_types, the longer the training time will be. Moreover, we can implicitly derive it by having high and low types in the evaluation set.